src/leaflike/db.clj

```diff
@@ -29,25 +31,33 @@
 
 (defn list-all
   []
-  (jdbc/query (db-spec) ["select * from bookmarks"]))
+  (jdbc/query (db-spec) (-> (select :*)
+                            (from :bookmarks)
+                            sql/format)))
 
-
-(defn list-by-id
+(defn list-by-params
   [params]
-  (jdbc/query (db-spec) ["select * from bookmarks
-                          where id = ?"
-                         (Integer/parseInt (:id params))]))
+  (jdbc/query (db-spec) (-> (select :*)
```

Comment:
Have a single `let` statement wherein you do the parsing for title and id (or any other params). Not a major change, it just keeps the code more readable by clubbing your bindings together.